Frontend/main.py

```diff
@@ -1,5 +1,5 @@
 import customtkinter as ctk
-
+import script
```

Comment:
Please use the below line before trying to import script from the parent directory as `ModuleNotFoundError` will occur
```
sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```